pkg/installation/installation.go

```diff
@@ -152,11 +159,11 @@ func (i *Installation) InstallKyma() (*Result, error) {
 	return result, nil
 }
 
-func (i *Installation) checkPrevInstallation() (string, error) {
+func (i *Installation) checkPrevInstallation(actionName string) (string, string, error) {
```

Comment:
Like in `InstallKyma()` this string parameter with lots of `if - else` in the function will be s source of problems.

Better to have 2 checks, one from an install perspective and another one from upgrade.